manager/views.py

```diff
@@ -1278,6 +1295,69 @@ def create_recipient_group_email_opens(request):
         )
     )
 
+def create_recipient_group_email_unopens(request, email_instance):
+    '''
+    Creates a recipient group based on emails
+    that did not open the email
+    POST only
+    '''
+    recipients = InstanceOpen.objects.filter(instance=email_instance).values_list('recipient__pk')
+
+    recipients_pks = [recipient[0] for recipient in recipients]
```

Comment:
Could we simplify this logic a big by passing `flat=True` to `.values_list()` above, instead of creating a new list to flatten the returned pks?

Reply:
Yep, we sure can. Updated